### PR TITLE
Adding an on-demand scrollbar for mobile off-canvas menu

### LINKED ISSUE #9383: [frio] Cannot scroll left fly-out sidebar on mobile
- [X] I have searched open and closed issues for duplicates

### Bug Description

When using Frio on mobile, when the fly-out sidebar is opened, it cannot scroll vertically. Any content in the sidebar beneath the edge of the screen is inaccessible.

### Steps to Reproduce

1. Switch your theme to Frio if it is not already.
2. Open Friendica on mobile or resize a desktop window to be narrow enough that the sidebar collapses and the three vertical dots (⋮) are visible on the left of the navbar.
3. Navigate to a page with a sidebar, such as the Contacts page or your Profile.
4.  Open the sidebar by tapping the (⋮) on the left of the navbar.
5. Attempt to scroll the navbar vertically.

**Actual Result:**

The navbar does not scroll. When you swipe vertically (or scroll with the scrollwheel if using a mouse), nothing moves.

**Expected Result:**

The navbar scrolls in accordance with the swiping or scrollwheel scrolling.

### Platform Info

**Friendica Version:** 2020.09

**Friendica Source:** https://files.friendi.ca/friendica-full-2020.09.tar.gz

**PHP version:** 7.3.19-1~deb10u1

**SQL version:** 10.3.23-MariaDB-0+deb10u1

### PR DESCRIPTION
**Pull request by vinzv**
This should fix #9383

### CODE REVIEW

File: view/theme/frio/css/style.css [hunk 1]
```diff
@@ -94,6 +94,7 @@ blockquote {
         height: 100%;
         padding-top: 100px;
         z-index: 10;
+	overflow: auto;
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: Please use the same indentation as the rest of the selector at least.

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Have you verified your fix to work?

**Comment by MrPetovan** [comment]
Funnily enough the length of your branch name is breaking GitHub’s mobile view:
BB4530F6-C4D6-4E28-9271-62182E9EDBA1

**Comment by vinzv** [comment]
> Have you verified your fix to work?

Yes, this is how it looks at /admin for example:

grafik

File: view/theme/frio/templates/jot.tpl [hunk 2]
```diff
@@ -111,6 +111,31 @@
...
+
+    $("#emojis").disMojiPicker()
+    $("#emojis").picker(emoji => console.log(emoji));
+    twemoji.parse(document.body);
+
+</script>
+
+<div id="emojis" style="display:none">
+</div>
+                                                <li role="presentation"><button type="button" class="btn-link" id="profile-emoji" onclick="myFunction();" title="{{$setloc}}"><i class="fa fa-smile-o" aria-hidden="true"></i></button></li>
```
Comment (MrPetovan) [inline view/theme/frio/templates/jot.tpl]: I believe this is work in progress code that shouldn't be part of this PR.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by MrPetovan** [comment]
> Yes, this is how it looks at /admin for example:

Is the low contrast expected?

**Review by MrPetovan — APPROVED** [review]

**Comment by vinzv** [comment]
Expected? Yes. It was like that before.
Wanted? No. But that's out of scope here.

**Pull request merged**